player/tests/data/buffer-copy.ron

```diff
@@ -1,5 +1,5 @@
 (
-    features: 0x1_0000,
+    features: 0x2_0000,
```

Comment:
I don't see why these had to change.

```suggestion
    features: 0x1_0000,
```

Reply:
@jinleili that change was from your initial PR, do you remember why these are needed ?

Reply:
Just to pass the test, 0x1_0000 will incorrectly match the newly added feature

Reply:
I found https://github.com/gfx-rs/wgpu/pull/3023/commits/b924a4d534429f21eca15a9d6078d48fcdf76cf7 which indicates that it should correspond to `MAPPABLE_PRIMARY_BUFFERS`.
Which was changed in https://github.com/gfx-rs/wgpu/pull/3534 to be on bit 34.
```diff
-        const MAPPABLE_PRIMARY_BUFFERS = 1 << 16;
+        const MAPPABLE_PRIMARY_BUFFERS = 1 << 34;
```

Reply:
I opened https://github.com/gfx-rs/wgpu/pull/4232 to take care of these.